perllib/Integrations/Echo.pm

```diff
@@ -197,7 +197,8 @@ sub FindPoints {
 sub GetServiceUnitsForObject {
     my $self = shift;
     my $id = shift;
-    my $obj = _id_ref($id, 'PointAddress');
+    my $type = shift;
+    my $obj = _id_ref($id, 'PointAddress', $type);
```

Comment:
Just a note, I added this so from the command line I could more easily look up by Uprn as well as Id :)